docs/tutorials.rst

```diff
@@ -20,3 +20,7 @@ Tutorials
     notebooks/mclp.ipynb
     notebooks/p-center.ipynb
     notebooks/p-median.ipynb
+    notebooks/lscpb.ipynb
```

Comment:
These notebooks must also reside in the [`docs/notebooks`](https://github.com/pysal/spopt/tree/main/docs/notebooks) folder.